### ISSUE TITLE
RANSAC could not find a valid consensus set. All `max_trials`

### ISSUE THREAD

**Issue by SSITB** [pycaret/pycaret#695]
While using compare model on regression problem

ValueError: RANSAC could not find a valid consensus set. All `max_trials` iterations were skipped because each randomly chosen sub-sample failed the passing criteria. See estimator attributes for diagnostics (n_skips*).

**Comment by pycaret** [comment]
@SSITB Hi, This is a known issue of the estimator. Can you try increasing `max_trials` param. For example:

```
create_model('ransac', max_trials = 300)
```

Let me know how it goes.

**Comment by Yard1** [comment]
`exclude=["ransac"]`

**Comment by koptimizer** [comment]
```
compare_models(
  exclude=['ransac']
  fold=10,
  round=4,
  sort='R2',
  n_select=1,
  budget_time=0,
  turbo=True,
  verbose=True,
)
```

**Comment by pycaret** [comment]
Just to add. In the next release `2.2` we are adding a new parameter to deal with errors in `compare_models`. It will automatically skip the model if an error is raised instead of stopping the entire function. 

Hope this helps.

**Comment by pycaret** [comment]
@SSITB 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.
